docs/concepts/dotnet-standard-library.md

```diff
@@ -0,0 +1,80 @@
+.NET Standard Library
+=====================
+
+It can be thought of as the next generation of [Portable Class Libraries](https://msdn.microsoft.com/library/gg597391.aspx), although is architecturally different. You can learn more 
```

Comment:
You also have to think that the first few sentences on a topic are used by search crawlers for SEO purposes so you should have a good introduction here with your main keywords such as .NET Standard Library.